src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -268,6 +268,18 @@ module sonic-acl {
 						range "0..1";
 					}
 				}
+
+				leaf BTH_OPCODE {
```

Comment:
Please add description for BTH_OPCODE and AETH_SYNDROME

Reply:
Added